charts-other/sidam/charts/sidam-web/values.yaml

```diff
@@ -0,0 +1,21 @@
+java:
```

Comment:
Do we need sidam-web? Not sure just trying to minimise our preview envs being overloaded :(512 mem and 250 cpu per image sounds a lot :()

Reply:
Yes, this is where user enters username and password and internally it points to API instance. We would not be able to reuse that frontend from AAT.